tools/functional-tester/etcd-tester/checks.go

```diff
@@ -46,9 +46,8 @@ func (hc *hashChecker) Check() (err error) {
 		hashes map[string]int64
 		ok     bool
 	)
-	for i := 0; i < 7; i++ {
-		time.Sleep(time.Second)
```

Comment:
move this sleep to the end of the for loop? we probably do not want to retry immediately.